pkg/summarizer/summary.go

```diff
@@ -522,6 +523,20 @@ func overallStatus(grid *statepb.Grid, recent int, stale string, brokenState boo
 	return summarypb.DashboardTabSummary_UNKNOWN
 }
 
+func allLinkedIssues(rows []*statepb.Row) []string {
+	issueMap := make(map[string]bool)
```

Comment:
Is `IssueSet` a better name? or `uniqueIssues` 

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_identifier_length

in his guidelines he recommended not including type in the name.

Reply:
I think issueSet helps describe what it's actually doing so using that!